velox/exec/Spiller.h

```diff
@@ -61,6 +61,7 @@ class Spiller {
       common::CompressionKind compressionKind,
       memory::MemoryPool* pool,
       folly::Executor* executor,
+      uint64_t maxSpillRunRows = 0,
```

Comment:
Can we require caller to pass maxSpillRunRows but for some constructor which always takes zero, then let's remove it from ctor? Thanks!